src/model/test/FloorspaceReverseTranslator_GTest.cpp

```diff
@@ -82,6 +82,35 @@ using namespace openstudio::model;
     OutputDebugString(os_.str().c_str()); \
   }
 
+void CompareSurfaceGroups(Model& model, Model& baseline) {
+
+  auto shadingGroup1 = model.getModelObjects<ShadingSurfaceGroup>();
+  auto shadingGroup2 = baseline.getModelObjects<ShadingSurfaceGroup>();
+  EXPECT_EQ(shadingGroup1.size(), shadingGroup2.size());
+  for (auto& shadingGroup : shadingGroup1) {
+    // Shading Surface Group names only match for building shading
+    if (shadingGroup.shadingSurfaceType() == "Building") {
+      auto match = baseline.getModelObjectByName<ShadingSurfaceGroup>(*shadingGroup.name());
+      EXPECT_TRUE(match.has_value());
```

Comment:
FYI it's possible to Improve reporting, for eg here:

```suggestion
      auto shadingGroupName = shadingGroup.nameString();
      auto match = baseline.getModelObjectByName<ShadingSurfaceGroup>(shadingGroupName);
      EXPECT_TRUE(match.has_value()) << "Could not locate shading group " << shadingGroupName;
```

Reply:
Thanks - yeah I sort of discovered that bay accident a while back so have been using it on other branches